### DISCUSSION TITLE
Grafana - Data lost after relogin

### DISCUSSION THREAD

**Discussion by dgawin** [WLAN-Pi/feedback#75] [Feature Request]
Hi,
i have a question about the Grafana Plugin in the WLAN-Pi (i got a R4).
If i use the Grafana Plugin and collect via Scanner the wireless, its working fine.
But if i close the browser and come back some seconds later, there is no Data stored and displayed. Only new data get collected after open the dashboard. Is it correct, or did i not complete the inital setup for this plugin?
Regards,
Dominik

**Comment by jiribrejcha** [comment]
Hi Dominik,

This is the expected behaviour. All data is streamed using the Grafana Line Protocol live to Grafana and stored in your browser cache. When browser window refreshes or gets closed, data will disappear. 

To retain data permanently, you could install Influx time-series database and edit data collection scripts (Data Streams) to this database.
